expresswrapper.js

```diff
@@ -326,20 +326,25 @@ function sendResponse (req, res, err, streamResponse, data, outMimeType = null)
 	}
 }
 
-function _request(method, route, streamResponse, callback)
+function _request(method, route, streamResponse, callback, options)
 {
-	if (method === "post" || method === "put" || method === "delete" || method === "get")
-	{
+	if ( method === "post" || method === "put" || method === "delete" || method === "get")
+	{	
 		this.app[method](route, (req, res) =>
 		{
 			try
 			{
+				if (!options || !options.acceptFiles) {
```

Comment:
@raphaelgoms this option should be documented at least as a comment.

Reply:
I documented in readme. The text is ok?

Reply:
Looks good to me, although an in code comment would suffice too

Reply:
Looking again at this, it seems to me, the need of this parameter is not making much sense as it introduces a seemingly unnecessary backward compatibility issue.